# This expresses an estimated pose with a reference coordinate frame and timestamp

std_msgs/Header header
geometry_msgs/PoseWithCovariance pose